README.md

```diff
@@ -33,6 +33,7 @@ _These assume you have `egghead-systems` running locally_
 - **Package management:** Yarn
 - **Scripts:** Yarn
 - **JS Flavor:** ES2016+ _(default `create-react-app`)_
+- **JS Utils:** Lodash
```

Comment:
It sure is handy! I've been using the array methods a lot more lately. Since we're using Rx and it is fully imported anyway, I wonder how much overlap there is in operators? I saw Ben Lesh say that one of the ways to start getting comfortable with Rx is to "use it like lodash" for these things.